pkg/didcomm/protocol/route/api.go

```diff
@@ -0,0 +1,13 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package route
+
+// Config provides the router configuration.
```

Comment:
Do we need this interface?
As I can see route service returns config in one of the exported function, but it can return struct instead. 
Unless we try to inject this config from somewhere.

Reply:
[Removed](https://github.com/hyperledger/aries-framework-go/compare/5234d05aa790ca34b1ede2f5f240e61d339640a2..94b9306c4274e5dbdd72571d2a46a7ff6dbf24e5#diff-829f87faf4556f4197c6a5292929cda5L10) the interface.